src/condor_starter.V6.1/io_proxy_handler.cpp

```diff
@@ -75,7 +75,16 @@ int IOProxyHandler::handle_request( Stream *s )
 	char line[CHIRP_LINE_MAX];
```

Comment:
It seems like we should be checking if the request is _over_ the limit
```suggestion
	char line[CHIRP_LINE_MAX+1];
```